email_alert_service/presenters/unpublishing_message_presenter.rb

```diff
@@ -0,0 +1,69 @@
+class UnpublishingMessagePresenter
+  EMAIL_DATE_FORMAT = "%l:%M%P, %-d %B %Y".freeze
+
+  def initialize(unpublishing_scenario, document, page_title)
+    @unpublishing_scenario = unpublishing_scenario
+    @document = document
+    @page_title = page_title
+  end
+
+  def call
+    [
+      "# #{page_title_markdown}",
+      ["Body:", page_title_markdown].join("\n"),
+      ["Change made:", unpublishing_scenario_note].join("\n"),
+      ["Time updated:", formatted_time].join("\n"),
+      footer,
+    ].join("\n\n")
+  end
```

Comment:
The message already has some of this: https://github.com/alphagov/email-alert-api/blob/main/app/builders/immediate_email_builder.rb#L40